.github/workflows/dev_pr/link.js

```diff
@@ -84,7 +84,7 @@ async function commentGitHubURL(github, context, pullRequestNumber, issueID) {
   const issueInfo = await helpers.getGitHubInfo(github, context, issueID, pullRequestNumber);
   const message = "* Closes: #" + issueInfo.number
   if (issueInfo) {
-    if (context.payload.pull_request.body.includes(message)) {
+    if (context.payload.pull_request.body != null && context.payload.pull_request.body.includes(message)) {
```

Comment:
How about simplifying this?

```js
    if (context.payload.pull_request.body && context.payload.pull_request.body.includes(message)) {
```

or

```js
const body = context.payload.pull_request.body || "";
if (body.includes(message)) {
  return;
}
await github.rest.pulls.update({..., body: body + "\n" + message});
```
```

Reply:
Thanks @kou ! I've applied the second suggested approach!